ui-tests/src/test/java/org/openmrs/reference/LoginTest.java

```diff
@@ -9,22 +9,20 @@
  */
 package org.openmrs.reference;
 
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.openmrs.reference.page.HomePage;
 
 import static org.junit.Assert.assertTrue;
 
-@Ignore
 public class LoginTest extends ReferenceApplicationTestBase {
     private HomePage homePage;
 
     @Test
     @Category(org.openmrs.reference.groups.BuildTests.class)
     public void verifyModulesAvailableOnHomePage() throws Exception {
         homePage = new HomePage(page);
-        assertPage(homePage);
+        assertPage(page);
```

Comment:
thanks @sharif as you continue working on this, just have a pointer here, why not assert the homePage since (page) has been passed to it